ui/app/models/identity/entity.js

```diff
@@ -46,6 +44,8 @@ let Model = IdentityModel.extend({
   canEdit: alias('updatePath.canUpdate'),
   canRead: alias('updatePath.canRead'),
   canAddAlias: alias('aliasPath.canCreate'),
+  policyPath: lazyCapabilities(apiPath`sys/policies`),
```

Comment:
used `lazyCapabilities` here instead of `attachCapabilities` below because they came in as `undefined` otherwise